### ISSUE TITLE
Increase spacebar trackpad speed

### ISSUE THREAD

**Issue by ghost** [FossifyOrg/Keyboard#36]
⁣

**Comment by ronniedroid** [comment]
I don't feel that it is slow at all.

Do you mean the fact that it moves one character at a time is slow? If that is the case, then adding a feature where it goes more than one character when the user goes faster would be the desired thing to do, but moving one character at a time should stay as is.

**Comment by Aga-C** [comment]
@ronniedroid If you compare it to other keyboards, e.g. SwiftKey, it is slow. It could even work like in SwiftKey, that while moving the finger slowly it goes one character at a time, but you can move your finger faster, and then it also scrolls faster. In SwiftKey it even enters the mode, that you can scroll up and down the multiline text area.

**Comment by ronniedroid** [comment]
@Aga-C yeah, I will see how easy would it be to make it go faster when the user scrolls faster, no promises though.

**Comment by Aga-C** [comment]
@naveensingh IMO, it's about slow movement. In Swiftkey, the speed of the cursor is dependent on how fast are you moving your finger over the space bar. In Simple Keyboard, the speed is constant, no matter how fast you move your finger.

**Comment by naveensingh** [comment]
Right, I just checked Swiftkey and the control is just better there but I don't notice this difference between Fossify and Google keyboard.

Google picks up the swipe gesture really quickly but the control isn't that great. It uses the long press event to launch language selection dialog whereas Fossify and Swiftkey both require a long press before activating cursor control.

**Comment by onegladius** [comment]
> Right, I just checked Swiftkey and the control is just better there but I don't notice this difference between Fossify and Google keyboard.
> 
> Google picks up the swipe gesture really quickly but the control isn't that great. It uses the long press event to launch language selection dialog whereas Fossify and Swiftkey both require a long press to activate cursor control. I like the way Swiftkey does it.

Yes this is what I meant

**Comment by naveensingh** [comment]
Okay

**Comment by naveensingh** [comment]
Since the author is now ghost, I have created a new issue for this: https://github.com/FossifyOrg/Keyboard/issues/129
